### ISSUE TITLE
Conversation StartOnEnter documentation inconsistent with behaviour

### ISSUE THREAD

**Issue by MarcusKohnert** [Vonage/vonage-dotnet-sdk#575]
**Describe the bug**
The documentation of the property [StartOnEnter](https://github.com/Vonage/vonage-dotnet-sdk/blob/a404398fb38d2808c60b9dcc9545b8a1c812e348/Vonage/Voice/Nccos/ConversationAction.cs#L74) says that `The default value of true ensures that [...]`.
If an instance of type `ConversationAction` is instantiated via the default constructor, the property `StartOnEnter` gets a value of `false` assigned (default value for boolean). This is an inconsistency between the documentation and the runtime behaviour.

This lead me to the scenario that I've created a conversation with attendees but the conversation never started because `StartOnEnter` was always false.

**To Reproduce**
```csharp
var action = new ConversationAction
{
    Name = "some-name"
};

new Ncco(action).ToString();
```

**Expected behaviour**
```json
[
    {
        "action": "conversation",
        "name": "some-name",
        "startOnEnter": true
    }
]
```
**Actual behaviour**
```json
[
    {
        "action": "conversation",
        "name": "some-name",
        "startOnEnter": false
    }
]
```

**Environment:**
 - .NET 8
 - SDK Version v7.2.0

**Additional context**
I guess that is an unwanted side effect of #463

**Comment by Tr00d** [comment]
Hi @MarcusKohnert,

Thanks for raising that one. It is indeed a nasty side effect of #463.

The default value isn't handled is in the SDK but it initially wasn't serialized in the JSON body when the default value was provided, delegating the decision of default value (`true`) to the API. 

I'll include that in the following release.

**Comment by Tr00d** [comment]
Hi @MarcusKohnert,

This has been fixed in the latest version, [v7.2.2](https://github.com/Vonage/vonage-dotnet-sdk/releases/tag/v7.2.2).

Let me know if it works out for you.

**Comment by Tr00d** [comment]
Closing issue after two weeks. Don't hesitate to reach out if yoou face any problem.
